README.md

```diff
@@ -18,7 +18,11 @@ npm install ipld-in-memory
 const IPLD = require('ipld')
 const inMemory = require('ipld-in-memory')
 
-const ipld = await inMemory(IPLD)
+// options object to be passed to the IPLD constructor - any
```

Comment:
More FYI. I understood as "any 'blockservice' property will be overridden" as "If you set options, then you also have to set the 'blockservice' property". Though I should read it as "This module sets the 'blockservice' itself and will ignore your value".

Reply:
I've tried to clarify the comment.  It should have been `blockService` instead of `blockservice` anyway..